### PR TITLE
build: upload binaries and docker images on deploy

### PR DESCRIPTION
**Pull request by adamdecaf**
Issue: https://github.com/moov-io/ach/issues/464

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -43,3 +43,20 @@ script:
 after_success:
 - bash <(curl -s https://codecov.io/bash)
 - make docker
+before_deploy:
+- make dist
+deploy:
+  provider: releases
+  api_key:
+    secure: TODO
```
Comment (wadearnold) [inline .travis.yml]: Do I need to do anything to allow for a token?

Comment (adamdecaf) [reply .travis.yml]: Oh, I need to remember how to encrypt the github token. Do you have one handy still?

**Comment by adamdecaf** [comment]
I think we'll have to merge this PR to see it attempt a deploy/release.

**Review by wadearnold — APPROVED** [review]

**Comment by wadearnold** [comment]
Let's try it.

**Pull request merged**